recipes/rust-activation/activate.bat

```diff
@@ -0,0 +1,9 @@
+set CARGO_HOME=%CONDA_PREFIX%.cargo.win
+set CARGO_CONFIG=%CARGO_HOME%\config
+set RUSTUP_HOME=%CARGO_HOME%\rustup
+set PATH=%CARGO_HOME%\bin:%PATH%
+
+if not exist "%CARGO_HOME%" mkdir "%CARGO_HOME%"
+
+echo [target.x86_64-pc-windows-msvc]>> %CARGO_CONFIG%
```

Comment:
This hardcoded value may conflict with a non default cxx_compiler. I may be preferable to require an explicit rust toolchain entry in cbc.yaml that is zipped with the compiler settings.

It's probably hard enough to get this right and enough of a corner case that you should ignore it for now.